Userland/Demos/Mandelbrot/Mandelbrot.cpp

```diff
@@ -118,8 +121,12 @@ class MandelbrotSet {
     void calculate_pixel(int px, int py, int max_iterations)
     {
         auto iterations = mandelbrot(px, py, max_iterations);
-        auto whole_iterations = floor(iterations);
-        auto partial_iterations = fmod(iterations, 1);
+        if (iterations == max_iterations) {
+            m_bitmap->set_pixel(px, py, Color::Black);
+            return;
+        }
```

Comment:
Did you implement that because the edge case was wrong, or because this is a fast path?
Except if this is closely related to the changes in `mandelbrot`, this should be in its own commit.

Reply:
Just a fast path. removed for now.